### PR TITLE
Capsules: Add an AirQuality capsule and a CCS811 Capsule

### PR DESCRIPTION
**Pull request by alistair23**
### Pull Request Overview

Add a AirQuality capsule which exposes AirQuality to userspace. Also add a CCS811 capsule which can interface with the CCS811 device via I2C.

We then connect them to the Apollo3 board and include tests

### Testing Strategy

Tock unit tests

### Documentation Updated

- [X] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [X] Ran `make prepush`.

### CODE REVIEW

**Review by bradjc — APPROVED** [review]

File: capsules/src/ccs811.rs [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+const RAW_DATA: u8 = 0x03;
+#[allow(dead_code)]
+const ENV_DATA: u8 = 0x05;
+#[allow(dead_code)]
+const NTC: u8 = 0x06;
+#[allow(dead_code)]
+const THRESHOLDS: u8 = 0x10;
+#[allow(dead_code)]
+const BASELINE: u8 = 0x11;
+const HW_ID: u8 = 0x20;
```
Comment (twilfredo) [inline capsules/src/ccs811.rs:31]: should this be `0x81` and used in line 219?

File: kernel/src/hil/sensors.rs [hunk 2]
```diff
@@ -75,6 +75,9 @@ pub trait AirQualityDriver<'a> {
 
 /// Client for receiving Air Quality readings
 pub trait AirQualityClient {
+    /// Called when the enviroment specify command has completed.
+    fn enviroment_specified(&self, result: Result<(), ErrorCode>);
```
Comment (twilfredo) [inline kernel/src/hil/sensors.rs:79]: typo on fn name should be`environment`

**Review by twilfredo — DISMISSED** [review]
LGTM!

**Review by phil-levis — CHANGES_REQUESTED** [review]
Please make the spelling fixes noted in the comments, then this is good to go.

File: capsules/src/ccs811.rs (hunk 1 above)
Comment (alistair23) [reply capsules/src/ccs811.rs:31]: This is the register address, `0x81` is the expected value at the address

**Comment by alistair23** [comment]
Ha! Fixed

**Review by twilfredo — APPROVED** [review]

**Review by phil-levis — APPROVED** [review]

**Pull request merged**